### PR TITLE
fix: sync up package versions

### PR DESCRIPTION
**Pull request by bryans99**

### CODE REVIEW

File: packages/embed-playground/package.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
     "@looker/embed-components": "*",
     "@looker/extension-utils": "^0.1.21",
     "@looker/redux": "^0.0.1",
-    "@looker/sdk": "^23.2.0",
-    "@looker/sdk-rtl": "^21.6.0",
+    "@looker/sdk": "^23.14.1",
+    "@looker/sdk-rtl": "^21.6.1",
```
Comment (jkaster) [inline packages/embed-playground/package.json:46]: If we set these to `*` does release-please update them for publishing? Or is this an extra lint/package dependency step we're going to need to add to our generation/publishing step? @treshugart would like your thoughts here

**Review by jkaster — APPROVED** [review]

File: packages/embed-playground/package.json (hunk 1 above)
Comment (bryans99) [reply packages/embed-playground/package.json:46]: To be honest I dont know. I'm just syncing the packages.

Comment (jkaster) [reply packages/embed-playground/package.json:46]: Understood. That's why I summoned Trey 😸

Comment (jkaster) [reply packages/embed-playground/package.json:46]: This is *not* a blocker for your PR, which I've approved.

**Pull request merged**